python/phonenumbers/phonenumberutil.pyi

```diff
@@ -0,0 +1,204 @@
+def _has_valid_country_calling_code(country_calling_code: int) -> bool: ...
+def format_number(numobj: PhoneNumber, num_format: int) -> str: ...
+def format_by_pattern(numobj: PhoneNumber, number_format: int, user_defined_formats: list[NumberFormat]) -> str: ...
+def format_national_number_with_carrier_code(numobj: PhoneNumber, carrier_code: str) -> str: ...
+def format_national_number_with_preferred_carrier_code(numobj: PhoneNumber, fallback_carrier_code: str) -> str: ...
+def format_number_for_mobile_dialing(numobj: PhoneNumber, region_calling_from: str, with_formatting: bool) -> str: ...
+def format_out_of_country_calling_number(numobj: PhoneNumber, region_calling_from: str) -> str: ...
+def format_in_original_format(numobj: PhoneNumber, region_calling_from: str) -> str: ...
+def _format_original_allow_mods(numobj: PhoneNumber, region_calling_from: str) -> str: ...
+def _raw_input_contains_national_prefix(raw_input: str, national_prefix: str, region_code: str) -> bool: ...
+def _has_formatting_pattern_for_number(numobj: PhoneNumber) -> bool: ...
+def format_out_of_country_keeping_alpha_chars(numobj: PhoneNumber, region_calling_from: str) -> str: ...
+def national_significant_number(numobj: PhoneNumber) -> str: ...
+def _prefix_number_with_country_calling_code(country_code: int, num_format: int, formatted_number: str) -> str: ...
+def _format_nsn(number: str, metadata: PhoneMetadata, num_format: int, carrier_code: str | None = ...) -> str: ...
+def _choose_formatting_pattern_for_number(available_formats: list[NumberFormat], national_number: str) -> NumberFormat | None: ...
+def _format_nsn_using_pattern(national_number: str, formatting_pattern: NumberFormat, number_format: int, carrier_code: str | None = ...) -> str: ...
+def example_number(region_code: str) -> PhoneNumber | None: ...
+def invalid_example_number(region_code: str) -> PhoneNumber | None: ...
+def example_number_for_type(region_code: str | None, num_type: int) -> PhoneNumber | None: ...
+def _example_number_anywhere_for_type(num_type: int) -> PhoneNumber | None: ...
+def example_number_for_non_geo_entity(country_calling_code: int) -> PhoneNumber | None: ...
+def _maybe_append_formatted_extension(numobj: PhoneNumber, metadata: PhoneMetadata, num_format: int, number: str) -> str: ...
+def _number_desc_by_type(metadata: PhoneMetadata, num_type: int) -> PhoneNumberDesc | None: ...
+def number_type(numobj: PhoneNumber) -> int: ...
+def _number_type_helper(national_number: str, metadata: PhoneMetadata) -> int: ...
+def _is_number_matching_desc(national_number: str, number_desc: PhoneNumberDesc | None) -> bool: ...
+def is_valid_number(numobj: PhoneNumber) -> bool: ...
+def is_valid_number_for_region(numobj: PhoneNumber, region_code: str) -> bool: ...
+def region_code_for_number(numobj: PhoneNumber) -> str | None: ...
+def region_code_for_country_code(country_code: int) -> str: ...
+def _region_code_for_number_from_list(numobj: PhoneNumber, regions: tuple[str, ...]) -> str | None: ...
```

Comment:
super-nit: `_region_code_for_number_from_list` should be one line up to match the `.py` file ordering.

Reply:
done